qcodes/dataset/data_set.py

```diff
@@ -1248,6 +1248,17 @@ def _load_to_xarray_dataset(self, data: ParameterData) -> "xr.Dataset":
 
         xrdataset.attrs["sample_name"] = self.sample_name
         xrdataset.attrs["exp_name"] = self.exp_name
+        xrdataset.attrs["guid"] = self.guid
+        xrdataset.attrs["run_timestamp"] = self.run_timestamp_raw
+        xrdataset.attrs["completed_timestamp"] = self.completed_timestamp_raw
+        xrdataset.attrs["parameters"] = self.parameters
```

Comment:
Parameters should redundant while run description is here? Or is it for convenience?

Reply:
Perhaps, I could see this go either way